test/common/expectPurchasabilityCodes.js

```diff
@@ -0,0 +1,21 @@
+const expect = require('./expect');
+const expectBignumber = require('./expectBignumber');
+
+const { hexToNumber } = web3.utils;
+
+const expectedMaximumLength = 10;
+
+module.exports = actualCodes => ({
+  toEqual: (expectedCodes) => {
+    expect(actualCodes.length).to.equal(expectedMaximumLength);
+
+    for (let i = 0; i < expectedCodes.length; i += 1) {
```

Comment:
```
expectedCodes.forEach((actualCodes) => {
  const actualCode = hexToNumber(actualCode);
      expect(actualCode).to.equal(expectedCode);
```
is more readable

Reply:
Actually it's a bit more complex than that. The actual array is of fixed length but the input (expected values) is of variable length. That's why I need to check for both expected values and following zeroes. I've refactored it a bit, how do you like it now?

Reply:
You're right, i thought is one array.

Hoverer still you can use .forEach method, second argument of cb is array index, so it should looks like:

```
expectedCodes.forEach((actualCodes, i) => {
  const actualCode = hexToNumber(actualCodes[i]);
      expect(actualCode).to.equal(expectedCode)
```